aws/services/ec2/resource.ftl

```diff
@@ -672,6 +672,49 @@
     ]
 [/#function]
 
+[#function getInitConfigSSHPublicKeys SSHPublicKeys linkEnvironment ignoreErrors=false priority=9 ]
+    [#local SSHPublicKeysContent = "" ]
+    [#list SSHPublicKeys as id,publicKey ]
+        [#if (linkEnvironment[publicKey.SettingName])?has_content ]
+            [#local SSHPublicKeysContent += linkEnvironment[publicKey.SettingName] + " " + id ]
+            [#if (SSHPublicKeys?keys)?seq_index_of(id) != ((SSHPublicKeys?keys)?size - 1)]
+                [#local SSHPublicKeysContent += "\n"]
+            [/#if]
+        [/#if]
+    [/#list]
+    [#return
+        {
+            "${priority}_scripts" : {
+                "files" :{
+                    "/home/ec2-user/.ssh/authorized_keys2" : {
```

Comment:
@roleyfoley , is it OK location for our additional file? Or should we store it somewhere under `/opt/codeontap` ? And what about the filename?  `authorized_keys2` has been deprecated, probably we should use something more specific?

Reply:
authorised keys are user based so I think its good to go into the ec2-user home drive. 
Maybe a better name for the file could be `authorized_keys_hamlet` just to show where the keys came from?